pkg/bindata/templates/profile.html

```diff
@@ -18,31 +18,143 @@ <h1><i class="fas fa-user-alt" style="float:left;font-size:40px"></i> User Profi
                                         </tr>
                                     </thead>
                                     <tbody>
+                                        <tr>
+                                            <td>
+                                                <a onClick="dialog($(this), 'left')" class="help-icon far fa-question-circle"></a>
+                                                <span class="dialogTitle">Auth Type</span>
+                                                <div style="display:none;" class="dialogText">
+                                                    Three auth types are currently supported.
+                                                    Auth Proxy and No Password are only available if the current request comes from a valid, known
+                                                    upstream IP address.
+                                                    <h4>Password: Local Username</h4>
+                                                    This is the default auth type.
+                                                    Select the Password option to log into this application with a configured username and password.
+                                                    Only 1 username and password is supported.
+                                                    When selecting this option, you must fill in the username and new password fields.
+                                                    If this is the current auth type you must also fill in the current password field.
+                                                    <h4>Auth Proxy: Header</h4>
+                                                    Select this option to log into this application using an HTTP header that container a username.
+                                                    This is commonly known as proxy authentication. We have
+                                                    <a href="https://notifiarr.wiki/en/Client/ReverseProxy">a wiki page</a>
+                                                    that provides instructions to configure Nginx for this feature.
+                                                    When selecting this option, you must also select a header for the auth proxy.
```

Comment:
```suggestion
                                                    When selecting this option, in addition to setting trusted upstreams, you must also select a header for the auth proxy.
```

Reply:
It's clear you haven't tried this yet. You can't select a header until you set trusted upstreams.